.github/workflows/ci.yml

```diff
@@ -80,16 +80,13 @@ jobs:
         run: make test
 
       - name: Upload cnd ${{ matrix.os }} binary
-        if: matrix.os == 'ubuntu-latest'
         uses: actions/upload-artifact@v1
         with:
           name: cnd
           path: target/debug/cnd
```

Comment:
You will need to add an identifier here. 
Github action will upload a binary called `cnd` for both now, macOs and Linux. This means it can't work because the slower one will overwrite the faster one. 

You will also need to download the correct one after wards.

Reply:
thanks! missed that :)